server.py

```diff
@@ -11,7 +11,7 @@
 
 if len(sys.argv) == 3:
     HOST=sys.argv[1]
-    PORT=sys.argv[2]
+    PORT=int(sys.argv[2])

```

Comment:
some python webservers (ie. gevent) complain if this value which is parsed as a string isn't then turned into an integer